Admin/AbstractAdmin.php

```diff
@@ -2382,6 +2398,13 @@ public function setBaseCodeRoute($baseCodeRoute)
      */
     public function getBaseCodeRoute()
     {
+        if ($this->isChild()) {
+            return $this->getParent()->getBaseCodeRoute().'|'.$this->getCode();
+        }
```

Comment:
BTW, I think I will have to use a lot of snippets like this one (`if ($this->getParent()->isChild())`). In `DefaultRouteGenerator::getCode(...)` for example.